package.json

```diff
@@ -3,8 +3,8 @@
   "version": "3.2.5",
   "private": true,
   "engines": {
-    "node": ">=16",
-    "npm": ">=8"
+    "node": ">=20",
+    "npm": ">=9"
```

Comment:
@apmatthews I ended up going with the `>=` syntax.

While it could make sense to explicitly peg the version to 20.x here, we are already doing that with .nvmrc.

Also, it is a better dev experience to be more permissive with newer versions of NPM (it's easy enough to use any version of Node with NVM, but switching NPM on the fly is not really a thing). So it makes sense to match the expectation of allowing newer versions of both Node and NPM.

Lastly, it simply feels sensible to match the style/syntax in [site-deploy](https://github.com/wpengine/site-deploy/blob/main/package.json).